modules/migration-using-cpma.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+// migration/migrating-openshift-3-to-4.adoc
+[id='migration-using-cpma_{context}']
+= Using CPMA
+
+.Prerequisites
+
+The following prerequisites apply to the {product-title} 3 cluster:
+
+* The cluster is updated to the latest synchronous release.
+* The environment health check has been run to confirm that there are no diagnostic errors or warnings.
+* The {product-title} version is 3.7, 3.9, 3.10, or 3.11.
+
+.Procedure
+
+. Log in to an {product-title} 3 cluster:
++
+----
+$ oc login https://<master1.example.com>
+----
+
+. Run CPMA. Each prompt requires you to provide input, as in the following example, which produces a report without the CR manifests:
++
+----
+$ ./cpma --manifests=false
```

Comment:
This seems to assume that you're in the directory you downloaded CPMA to. I'd add a step about navigating to that directory.

Reply:
No, it doesn't. `./cpma` is an absolute path, so you can run it from anywhere.

Reply:
OK, It's just an rpm, so I'll add a yum install command. It will create the /cpma directory.

Reply:
Is "cpma" a directory or a command? The way it's used above, it's an executable file.

And the "." in "./cpma" means "in the current directory", so it's a relative path.